Dockerfile

```
@@ -1,13 +1,15 @@
-FROM golang:1.23 as build
+FROM golang:1.25 as build
 WORKDIR /app
 COPY . .
 ENV GOOS linux
 ENV CGO_ENABLED 0
 RUN go build -v -o app ./cmd/ledger
+RUN go build -v -o healthcheck ./cmd/healthcheck
 
 # Copy the binary to distroless image
 FROM gcr.io/distroless/base as prod
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)